src/plugins/intel_npu/tests/functional/CMakeLists.txt

```diff
@@ -11,6 +11,14 @@ set(EXCLUDED_FUNC_TESTS_DIR "")
 set(OPTIONAL_FUNC_TESTS_INCLUDES "")
 set(OPTIONAL_FUNC_TESTS_LIBS "")
 
+list(APPEND OPTIONAL_FUNC_TESTS_INCLUDES
+    "${OpenVINO_SOURCE_DIR}/src/plugins/intel_npu/src/compiler/include"
+)
+
+list(APPEND OPTIONAL_FUNC_TESTS_LIBS
```

Comment:
adapter is still optional under ENABLE_DRIVER_COMPILER_ADAPTER 
so this should break a build without adapter

Reply:
Good point! Fixed.